modules/samples/artifacts/PublishHttpOAuthRequest/PublishHttpOAuthRequest.siddhi

```diff
@@ -0,0 +1,72 @@
+@App:name("PublishHttpOAuthRequest")
+@App:description("Send an OAuth events via HTTP transport")
```

Comment:
We are not sending "OAuth events" here. We are sending HTTP events to an OAuth-protected endpoint. Let's fix the descrpition here and in 'Purpose' section.